src/api/groupByService.ts

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react'
+import { BASE_PATH } from './apiConfig'
+import agencyList from 'open-bus-stride-client/agencies/agencyList'
+
+/*
+example response
+[
+  {
+    "gtfs_route_date": "2023-02-27",
+    "operator_ref": 0,
+    "day_of_week": "string",
+    "total_routes": 0,
+    "total_planned_rides": 0,
+    "total_actual_rides": 0
+  }
+]
+*/
+export type GroupByResponse = {
+  gtfs_route_date: string
+  operator_ref: number
+  day_of_week: string
+  total_routes: number
+  total_planned_rides: number
+  total_actual_rides: number
+}[]
+
+async function groupby({
```

Comment:
I like the convention of adding async suffix to async methods

Reply:
If there is both non-async and async method that returns the same thing I suffix the async one with Async. Otherwise not.
Is this rule important to you?
we can add the Async suffix, despite it's not necessary

Reply:
its important cause in js/ts you can easily not `await` an async by mistake, I find this convention helps that, and also helps with being aware this is not a "simple" method call